docs/data/destinations/google-analytics-4-firebase.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Google Analytics 4 (iOS/Android) Event Streaming
+description: Amplitude Data's Google Analytics 4 integration enables you to stream your Amplitude event data straight to Google Analytics with just a few clicks.
+---
+
+Amplitude Data's Google Analytics 4 (iOS/Android) integration enables you to stream your Amplitude event data straight to Google Analytics with just a few clicks.
+
+## Considerations
+
+The Google Analytics 4 (iOS/Android) destination works with an iOS or Android mobile application using Firebase. If you are working with a web application instrumented with Google Tag (gtag.js), set up a [Google Analytics 4 (Web)](../google-analytics-4-gtag) destination instead.
+
+- Amplitude sets `non_personalized_ads` to `true` for all sent events.
+- Google Analytics 4 property names are composed of alphanumeric characters and underscores. Amplitude replaces white spaces in property names with a single underscore and strips all other non-alphanumeric characters from the names.
+- Amplitude sends custom events using Amplitude `event_type` as event name.
+
+## Setup
+
+### Prerequisites
+
+To set up event streaming to GA4, you need the following: 
+
+- A `Firebase App ID` 
+- A `Measurement Protocol API secret`
+- A custom user property in Ampltiude containing the Firebase app instance ID for each user
+
+To find these values:
+
+1. Open the [Google Analytics home page](https://analytics.google.com/analytics/web). 
+2. Click the **gear icon** to open Admin, then select **Data Streams**. 
+3. Pick the data stream to stream the Amplitude events to.
+   - `Measurement Protocol API secret` is available under the `Additional Settings` section, by opening `Measurement Protocol API secrets`. You can create a new API secret that's specific to this use case, that you can manage and remove independently the others. Copy the value in the `Secret value` column of the appropriate row.
+4. Navigate to your Firebase console.
+5. You can find your `Firebase App ID` under `Projects Settings > General > Your Apps > App ID`.
+
+### Amplitude setup 
+
+1. In Amplitude, navigate to **Data**.
+2. On the sidebar, under **Connections**, select **Catalog**.
+3. Select the **Destinations** tab.
+4. Select **Event Streaming**.
+5. Click **Google Analytics 4 (Web)**.
+2. Enter a sync name, then click **Create Sync**.
+
+#### Configure settings
+
+1. On the **Settings** tab, click **Edit**.
```

Comment:
Didn't realize that was only on staging which is what I must have been referencing. We're currently fixing a UI bug right now so that'll be deployed first.